browser/model/devstudio.js

```diff
@@ -11,7 +11,7 @@ import JdkInstall from './jdk-install';
 import Platform from '../services/platform';
 
 class DevstudioInstall extends InstallableItem {
-  constructor(keyName, installerDataSvc, targetFolderName, downloadUrl, fileName, sha256sum, additionalLocations, additionalIus, useDownload) {
+  constructor(keyName, installerDataSvc, targetFolderName, downloadUrl, fileName, sha256sum, additionalLocations, additionalIus, useDownload, defaultOption = 'install') {
```

Comment:
Wouldn't it be nicer if defaultOption was moved to InstallableItem and loaded from its requirement? With install being the default if the requirement has none specified of course.